src/Library/demos/Email/main.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "Email",
+  "category": "platform",
+  "description": "Compose an email message",
```

remember to write description from the perspective of the developer

```suggestion
  "description": "Trigger an email message",
```